src/components/Button.jsx

```diff
@@ -28,19 +30,23 @@ const Core = function (
 
   if (display === 'back') {
     return (
-      <Button
-        size="small"
-        startIcon={<BackIcon />}
-        disabled={disabled}
-        style={{
-          padding: '4px 16px',
-          ...style,
-        }}
-        ref={ref}
-        {...rest}
-      >
-        {children}
-      </Button>
+      <Tooltip title={tooltiptext}>
```

Comment:
I know this makes things more complicated, but the Tooltip component shouldn't be rendered unless it is being used